.github/workflows/test.yml

```
@@ -11,6 +11,42 @@ permissions:
+          sudo apt-get update -qq
+          sudo apt-get install -y -qq shellcheck
+          pip install --quiet ruff
+
+      - uses: actions/setup-node@v7
+        with:
+          node-version: '24'
+
+      # Gate at error+warning severity (info-level findings are not blockers).
+      # SC1091: shellcheck cannot follow dynamic source paths — expected.
+      - name: shellcheck (scripts + hooks)
+        run: shellcheck --severity=warning --exclude=SC1091 scripts/*.sh hooks/*.sh *.tmux
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2035:info:1:72: Use ./*glob* or -- *glob* so names with dashes won't become options

(shellcheck)